frontend/src/components/layout/Footer.tsx

```diff
@@ -148,6 +148,12 @@ const Footer = () => {
             >
               {t("about")}
             </Link>
+            <Link
+              href="/consultants"
+              className="flex h-12 items-center justify-center text-xl text-inherit hover:underline sm:block sm:h-auto sm:text-sm"
+            >
+              {t("consultants")}
```

Comment:
needs to be added to the `en` `common.json`

Reply:
Pushed a new version that I think does the right thing